playbooks/roles/memcached/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+- name: Open memcached port on the firewall
+  ufw:
+    rule: allow
+    port: "{{ item }}"
+    proto: tcp
+  with_items:
```

Comment:
`with_items`: this is copy+pasted from other roles that use it, also with some port. It's not necessary but it doesn't matter. I changed it to only 1 item and added a commit.

I don't think we need to protect it, because these roles are used when we want to offer servers to the outside. That's what other roles do, like [mysql](https://github.com/open-craft/ansible-playbooks/blob/c2d335e47dc57cee95261c51d0033c3fbaf4e577/playbooks/roles/mysql/tasks/main.yml#L40), [mongodb](https://github.com/open-craft/ansible-playbooks/blob/c2d335e47dc57cee95261c51d0033c3fbaf4e577/playbooks/roles/mongodb/tasks/main.yml#L46), [elasticsearch](https://github.com/open-craft/ansible-playbooks/blob/c2d335e47dc57cee95261c51d0033c3fbaf4e577/playbooks/roles/elasticsearch/tasks/main.yml#L44): they cause the port to be opened, without any gating in their host_vars ([e.g.|https://github.com/open-craft/ansible-secrets/blob/d498696cef86f05850e34040e8112e4434dae638/host_vars/mysql.opencraft.hosting.yml]). In addition, even if someone accidentally includes this role thinking it's needed for internal memcached servers, the port won't be opened to the Internet because Ubuntu's memcached listens by default at `127.0.0.1` (you explicitly need to override it to be `0.0.0.0`).

Reply:
Ok fair enough :+1: